src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -53,3 +53,26 @@
      handle-connection)))
 
 (rf/reg-event-fx :wallet/get-key-pair-export-connection get-key-pair-export-connection)
+
+(defn wallet-validate-seed-phrase
+  [_cofx [seed-phrase on-success on-error]]
+  {:fx [[:multiaccount/validate-mnemonic [seed-phrase on-success on-error]]]})
+
+(rf/reg-event-fx :wallet/validate-seed-phrase wallet-validate-seed-phrase)
+
+(defn make-seed-phrase-keypair-fully-operable
+  [_ [mnemonic password on-success]]
+  {:fx [[:json-rpc/call
+         [{:method     "accounts_makeSeedPhraseKeypairFullyOperable"
+           :params     [(-> mnemonic security/safe-unmask-data)
+                        (-> password security/safe-unmask-data native-module/sha3)]
+           :on-success (fn []
+                         (rf/dispatch [:wallet/get-accounts])
+                         (rf/dispatch [:wallet/get-keypairs])
+                         (when (fn? on-success)
+                           (on-success)))
+           :on-error   #(log/info "failed to make key pair fully operable with seed phrase"
+                                  {:error %
+                                   :event :wallet/make-seed-phrase-keypair-fully-operable})}]]]})
```

Comment:
Here we're defining an event handler for creating an effect that will call the status-go accounts API function `MakeSeedPhraseKeypairFullyOperable`.

Reply:
Perhaps we need a `on-error` function to be passed as well 🤔

Reply:
IMO using `->` for only one step is not a good idea, when I see `->` I usually expect some transformations for the first parameter (e.g `(-> value f g h)`) but when I see `->` only with one transformation I think it's simpler if it were `(f value)` instead of `(-> value f)`)

Reply:
Haha yeah I think this was me being captivated by the aesthetics 😄 
I think it makes sense to change it 👍